srcpkgs/fcitx-hangul/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'fcitx-hangul'
+pkgname=fcitx-hangul
+version=0.3.1
+revision=1
+build_style=cmake
+depends="fcitx>=4.2.9 libhangul"
+makedepends="libhangul-devel fcitx-devel intltool pkg-config"
+short_desc="Hangul (Korean) support for fcitx"
+homepage="https://github.com/fcitx/fcitx-hangul"
+license="GPL-2.0"
```

Comment:
`GPL-2.0-or-later`

Reply:
No version of the SPDX identifiers were initially stated, although they're both valid.